Looks like it failed to pass the `black` linting step, although I can't see what it's complaining about. All that seems to be shown in the build log is:

`##[error]Bash exited with code '1'.`

https://dev.azure.com/home-assistant/Home%20Assistant/_build/results?buildId=6983&view=logs&j=2fa2c639-d05e-530d-034e-4fb413fcea4e&t=d5feafd4-bcc8-556b-1e3d-b40f145aa3be&l=12

--- a/homeassistant/components/venstar/climate.py
+++ b/homeassistant/components/venstar/climate.py
@@ -268,7 +268,8 @@ class VenstarThermostat(ClimateDevice):
                 success = False
                 _LOGGER.error(
                     "The thermostat is currently not in a mode "
-                    "that supports target temperature: %s", operation_mode
+                    "that supports target temperature: %s", 
+                    operation_mode,
                 )
 
             if not success:

--- a/homeassistant/components/venstar/climate.py
+++ b/homeassistant/components/venstar/climate.py
@@ -268,7 +268,8 @@ class VenstarThermostat(ClimateDevice):
                 success = False
                 _LOGGER.error(
                     "The thermostat is currently not in a mode "
-                    "that supports target temperature: %s", operation_mode
+                    "that supports target temperature: %s",
+                    operation_mode,
                 )
 
             if not success:

